docs/pages/includes/device-trust/troubleshooting.mdx

```diff
@@ -54,7 +54,8 @@ follow the steps below:
 1. Make sure your device is [registered and enrolled](
    ../../access-controls/device-trust/device-management.mdx#register-a-trusted-device)
 2. Install [Teleport Connect](
-   ../../connect-your-client/teleport-connect.mdx#installation--upgrade)
+   ../../connect-your-client/teleport-connect.mdx#installation--upgrade).
+   Use the DEB or RPM packages on Linux.
```

Comment:
I think we should be more explicit here. If the tarball doesn't work, let's say so and explain why.

Reply:
Done. I think it's a bit awkward, so let me know if you have any suggestions.